### DISCUSSION TITLE
How to disable fullscreen？

### DISCUSSION THREAD

**Discussion by atopx** [bevyengine/bevy#10219] [Q&A] [answered]
I hope it's a fixed size window, what should I do.

**Comment by atopx** [comment]
remarks：Desktop game, and bevy version is 0.11

**Comment by SIGSTACKFAULT** [comment]
it should be windowed by default; can I see your main function, specifically `app.add_plugins` calls?

**Comment by NiklasEi** [comment]
I think you are looking for this window setting https://github.com/bevyengine/bevy/blob/38e0a8010e57b872b7ab33fdf38d347db183907c/crates/bevy_window/src/window.rs#L129

**Reply by atopx** [reply]
The `resizeable` property cannot disable the maximize button in the upper right corner of the window. Here is my code:
```rust
// init windows plugin
        .add_plugins(DefaultPlugins.set(WindowPlugin {
            primary_window: Some(Window {
                title: public::WIN_TITLE.to_string(),
                resolution: (WIN_SIZE.w, WIN_SIZE.h).into(),
                mode: WindowMode::Windowed,
                resizable: false,
                ..default()
            }),
            ..WindowPlugin::default()
        }).set(
            // log plugin
            LogPlugin {
                level: bevy::log::Level::TRACE,
                ..LogPlugin::default()
            })
        )
        .run()
}
```

**Comment by mockersf** [comment, marked as answer]
Control over the maximize button will be exposed in Bevy 0.12 (which should be a few days away), but it is not possible in Bevy 0.11
See the window_settings example: https://github.com/bevyengine/bevy/blob/afe8b5f20da2bfa9629fa8ed1ff24852553e616c/examples/window/window_settings.rs#L24-L27
